conda-recipe/vcdat/meta.yaml

```diff
@@ -20,4 +20,4 @@ source:
   git_rev: v0.0.1
```

Comment:
instead of `version: 0.0.2` you want `version: master`? I can make that change in the PR.